doc/reference/reference_lua/net_box.rst

```diff
@@ -163,54 +163,91 @@ Below is a list of all ``net.box`` functions.
     such as select, update or delete.
     
     Possible options:
+    
+    *   ``user/password``: you have two ways to connect to a remote host: 
+        using :ref:`URI <index-uri>` or using the options `user` and `password`. For 
+         example, instead of ``connect('username:userpassword@localhost:33301')`` 
+         you can write ``connect('localhost:33301', {user = 'username', password='userpassword'})``.
```

Comment:
```suggestion
         example, instead of ``connect('username:userpassword@localhost:3301')`` 
         you can write ``connect('localhost:3301', {user = 'username', password='userpassword'})``.
```
Although you can use virtually any port, 3301 is the default.

Reply:
Fixed.